types/kos-core/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"interface-name" : [true, "always-prefix"]
```

Comment:
New definitions for package cannot contain anything apart from extends clause in tslint json.

Reply:
Hi, sheetalkamat, Thanks for your commit. I have removed the config item, could you please check it again?